blockchain-source/src/cardano/block.rs

```diff
@@ -74,6 +88,45 @@ impl Block {
             Block::Block4(blk) => &blk.tx_bodies,
             Block::Alonzo(blk) => &blk.tx_bodies,
             Block::Vasil(blk) => &blk.tx_bodies,
+            Block::Conway(blk) => &blk.tx_bodies,
         }
     }
 }
+
+/// Transaction Body
+#[derive(Clone, Debug, CborRepr, PartialEq, Eq)]
+#[cborrepr(structure = "mapint", skipkey = 10, skipkey = 12)]
+pub struct TransactionBody {
```

Comment:
had to do this for the transaction body to get parsed, otherwise it fails parsing the certs and the transactions that have the last 4 fields.